vispy/app/backends/ipython/_widget.py

```diff
@@ -4,7 +4,7 @@
 
 try:
     from ipywidgets.widgets import DOMWidget, register
-    from traitlets import Unicode, Int, Bool
+    from traitlets import Unicode, Int, Bool, Dict
```

Comment:
Before I forget, `Dict` will need to be added to the exception block below in case someone tries to import this but doesn't have things installed. It gives a more logical error iirc.

Reply:
Oops, of course! My mistake.